config/dimensions.yaml

```diff
@@ -1,6 +1,6 @@
-dataUnits:
+dimensions:
   version: 0
-  dimensions:
+  units:
```

Comment:
I really struggled with this.  It really ought to be "dimensions" for both the outer (as the name of the system) and the inner key (as the name of the central concept in that system), but that confuses some of our "guess what level this config is at" logic.  Without "Dimension", "graph" or "universe" doesn't make sense at the top level, I think.

Maybe I should capitalize the second-level "Dimensions", and also change "joins" to "DimensionJoins", to exactly match the classes, and rely on capitalization to avoid confusing the config code?

Reply:
Call top level `DimensionGraph`/`DimensionUniverse`? Case-only difference in names is generally considered questionable style.

Reply:
I'm going to look into merging the contents of `units` and `joins` into `elements`, and see if that's at least no worse in terms of constructing the universe `DimensionGraph`.  It feels a little ugly to make that kind of change to just work around a naming problem, but given that `Dimension` and `DimensionJoin` now have more in common, it might be better overall (and if it isn't I'll go back to thinking more about names).

Reply:
Success!  Moving everything under an `elements` heading does actually simplify the graph building logic, so it's worth keeping, and this naming issue is no more.